moz_generic/tests/hindsight/run/output/d2p.cfg

```diff
@@ -0,0 +1,29 @@
+filename        = "s3_parquet.lua"
+message_matcher = "Type == 'moz_generic' && Fields[namespace] == 'generic' && Fields[docType] == 'test' && Fields[docVersion] == '1'"
+ticker_interval = 60
+preserve_data   = true
+
+parquet_schema = [=[
```

Comment:
Technically all the input specific parquet schemas would have already been tested here https://github.com/mozilla-services/mozilla-pipeline-schemas/blob/master/tests/hindsight/run/output/test_parquet_schemas.lua what is the point of writing out a generic parquet file?  Wouldn't we want a generic_s3_parquet output?